core/src/scss/utilities/mixins/link/_more-link.scss

```diff
@@ -1,16 +0,0 @@
-@charset "UTF-8";
-
-//
-// @more-link()
-//
-// Default more link styling, with raquo.
-//
-// Style guide: Mixins.Link.more-link
-//
-@mixin more-link {
```

Comment:
Instead of completely removing the mixin, please deprecate it and patch through to the new option.

Reply:
Done